azure-pipelines.yml

```diff
@@ -131,6 +131,17 @@ jobs:
       ninja -C build-verilator all
       ninja -C build-fpga all
     displayName: 'Build embedded targets'
+  # XXX: We do not package build-fpga, only build-verilator, since the results of this
+  # job are only used for testing, not distribution. Eventually, only this job will be
+  # used for distribution.
```

Comment:
Can't we use the make-generated artifacts for testing? This would allow us to keep this PR focused on getting the Verilator-based CI up and running, and making switching to another way of generating the software a separate step. And it ensures that we test what we actually ship.